### DISCUSSION TITLE
Strange request

### DISCUSSION THREAD

**Discussion by 1979139113** [projectdiscovery/httpx#647] [Q&A] [answered]
command
```
echo gm7.org | ./httpx -title -random-agent -threads 100 -rate-limit 200 -x GET -timeout 5 -stats -fr -p https:443,80 -nc
```
then monitor through `ncat`

<img width="817" alt="图片" src="https://user-images.githubusercontent.com/24983597/171338950-d353555b-2646-4a86-9ffb-878bd2eedd97.png">

but what's this?

another picture

<img width="1293" alt="图片" src="https://user-images.githubusercontent.com/24983597/171341162-b8c70a43-c681-44e1-b87e-875b7123033d.png">

**Comment by ehsandeep** [comment]
@1979139113 probably you are trying to connect `https:443,80`? do you see the same with `http:443,80` request?

**Reply by 1979139113** [reply]
> @1979139113 probably you are trying to connect `https:443,80`? do you see the same with `http:443,80` request?

same

<img width="1858" alt="图片" src="https://user-images.githubusercontent.com/24983597/172303416-53ff844a-3b02-4a9a-af44-4b0128437c20.png">

<img width="1411" alt="图片" src="https://user-images.githubusercontent.com/24983597/172303448-6cdec008-2aad-4485-aae4-91b7b0a9c509.png">

**Comment by 1979139113** [comment, marked as answer]
httpx will scan http and https protocols by default, if you specify the port 80 as http through the parameter ` -p http:80` , its ok, no strange request
